sdks/go/pkg/beam/runners/universal/universal.go

```diff
@@ -40,6 +41,18 @@ func init() {
 
 // Execute executes the pipeline on a universal beam runner.
 func Execute(ctx context.Context, p *beam.Pipeline) error {
+	if !beam.Initialized() {
+		log.Warn(ctx, "Beam has not been initialized. Call beam.Init() before pipeline construction.")
```

Comment:
Portable runners require beam.Init to function at all. It's reasonable to hard fail here instead of just logging.
The direct runner is a special case, since the pipeline is executed locally before the function returns.

Reply:
Done, changed it to a panic instead.